python/ray/util/xgboost/simple_tune.py

```diff
@@ -60,13 +75,17 @@ def main():
             "extra_cpu": num_actors * num_cpus_per_actor
         })
 
-    # Load the best model checkpoint
-    import xgboost as xgb
-    import os
-
     # Load in the best performing model.
-    best_bst = xgb.Booster()
-    best_bst.load_model(os.path.join(analysis.best_logdir, "model.xgb"))
+    # If using Ray Client, make sure model loading happens on server.
```

Comment:
maybe you can add a comment here about how one would use the ray client. 

Also I think perhaps the cleanest way to do this may be to have a commented out section for the "client.ray.is_connected" block, since the client isn't used by default in this example

Reply:
Yes that's a good point, I will comment it out.

I'll also make sure to add this in the "Best Practices" documentation.